File: Assignment__14/Question5.c
//Write a program which accept number from user and count frequency of such a digits which are less than 6. 

#include<stdio.h>
#include<stdbool.h>

int Display(int iNo)
{
    int iDigit = 0;
    int FreqTwo = 0;

    if(iNo < 0)
    {
        iNo = -iNo;
    }

    while(iNo !=0)
    {
        iDigit = iNo % 10;

        if(iDigit < 6) 
        {                       
            FreqTwo = FreqTwo + 1;
        }                       
        iNo = iNo / 10;  
    }

    return FreqTwo;
}

int main()
{
    int iValue = 0;
    int iResult = 0;

    printf("Enter the number : \n");
    scanf("%d", &iValue);

    iResult = Display(iValue);

    printf("The number of two's are : %d\n", iResult);
    
    return 0;
}

File: Assignment__16/Question5.java
//Write a program to count the number of digits in a given number.

import java.util.Scanner;

class logic
{
    int DisplayCount(int iNo)
    {
        int iDigit = 0;
        int iCount = 0; 

        while(iNo != 0)
        {
            iDigit = iNo % 10;
            iCount = iCount + 1;
            iNo = iNo / 10;
        }

        return iCount;
    }
}

class Question5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        int iFrequency = 0;

        logic lobj = new logic();
        iFrequency = lobj.DisplayCount(iValue);

        System.out.println("The number of digits are " + iFrequency);

    }    


} 
